app/views/partials/foyer/ressources/types.html

```diff
@@ -1,16 +1,15 @@
 <h1>{{ pageTitle }}</h1>
 
 <p>
-  Sélectionnez les ressources perçues <strong>entre {{ debutAnneeGlissante }} et {{ finAnneeGlissante }}</strong>,
+  Sélectionnez les ressources perçues <strong>depuis {{ debutAnneeGlissante }}</strong>,
   vous pourrez ensuite saisir les montants.
 </p>
 
 <form method="post" name="form" novalidate ng-submit="submit()">
   <div class="ressource-types-block">
     <div class="capture-{{ category.id }}" ng-repeat="category in ressourceCategories">
-      <div class="capture-title">
-        <h2>{{ category.label }}</h2>
-      </div>
+      <details>
+        <summary class="capture-title">{{ category.label }}</summary>
```

Comment:
I gave a last shot to bootstrap's `accordion` but it would need big changes on the style sheet that may be not worth it.